lib/shared/store.js

```diff
@@ -36,7 +36,9 @@ const packageJSON = require('../../package.json')
  */
 const DEFAULT_STATE = Immutable.fromJS({
   availableDrives: [],
-  selection: {},
+  selection: {
+    drives: new Immutable.OrderedSet()
```

Comment:
For now it needs to be ordered, and maybe we'd prefer/require ordered sets later. This is due to it pretending to be single-selection while we're getting multi-writes in fully.

Reply:
> maybe we'd prefer/require ordered sets later

The reason I'd be :-1: on an OrderedSet is because the user has no way (without unselecting everything again) of changing that ordering.